positions/senior-libs-developer.md

```diff
@@ -1,5 +1,5 @@
 # Senior Library Developer
-Hi :) We are looking for an experienced libraries developer to design and implement high-performance data processing libraries for Enso (https://enso.org, YC S21), a functional, hybrid visual/textual programming language with immutable memory. You will be working in an incredibly talented team with Edward Kmett (chair of the Haskell libraries committee), Jaroslav Tulach (founder of NetBeans, co-creator of GraalVM Truffle), and many more.
+Hi, We are looking for an experienced libraries developer to design and implement high-performance data processing libraries for Enso ([ENSO.org](https://enso.org), YC S21), a functional, hybrid visual/textual programming language with immutable memory. You will be working in an incredibly talented team with Edward Kmett (chair of the Haskell libraries committee), Jaroslav Tulach (founder of NetBeans, co-creator of GraalVM Truffle), and many more.
```

Comment:
small letter after a comma, also link this way looks strange in rich diff

Reply:
ok